src/test/ui/const-generics/infer/uninferred-consts.min.stderr

```diff
@@ -3,6 +3,11 @@ error[E0282]: type annotations needed
    |
 LL |     Foo.foo();
    |         ^^^ cannot infer the value of const parameter `N` declared on the associated function `foo`
+   |
+help: consider specifying a const value in the function call
+   |
+LL |     Foo.foo::<N>();
```

Comment:
Could you add a new example to this file that requires multiple const arguments to be specified?